drivers/mcp2515/mcp2515_defines.h

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mcp2515
+ * @brief       Driver for the Microchip MCP2515 can controller.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Defines for the MCP2515 can controller driver.
+ *
+ *  MCP2515 SPI CAN Controller Register & Configuration Constant
+ *
+ * @author      Toon Stegen <toon.stegen@altran.com>
+ */
+#ifndef MCP2515_DEFINES_H
+#define MCP2515_DEFINES_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * User configuration
+ * @{
+ */
+#define CAN_SPI_CS_PORTBIT BIT4
+#define CAN_SPI_CS_PORTOUT P2OUT
+#define CAN_SPI_CS_PORTDIR P2DIR
+
+#define CAN_IRQ_PORTBIT BIT3
+#define CAN_IRQ_PORTOUT P1OUT
+#define CAN_IRQ_PORTDIR P1DIR
+#define CAN_IRQ_PORTREN P1REN
+#define CAN_IRQ_PORTIES P1IES
+#define CAN_IRQ_PORTIE P1IE
+#define CAN_IRQ_PORTIFG P1IFG
+/** @} */
+
+/**
+ * BoosterPack contains 8MHz crystal w/ 22pF load caps
+ */
+#define CAN_OSC_FREQUENCY 8000000
```

Comment:
Does this parameter relate to [this one](https://github.com/RIOT-OS/RIOT/pull/13624/files#diff-a1ae72b8445f0dcf7050c4b06164bd92R61)? Do we need it? If so, it should be prefixed with `MCP2515_`

Reply:
Your "this one" link doesn't seem to work on my side, but I'm assuming you're talking about `MCP2515_CLK`. I think they are related, yes. I think the other should be used over this one. Removing.